ColiseuPlayerTests/ColiseuPlayerTests.swift

```diff
@@ -113,7 +114,7 @@ class ColiseuPlayerTests: XCTestCase {
 
     func testPlaySongIndexIsInvalidIndex() {
         // given
-        let expectedResult = true
+        let expectedResult = false
```

Comment:
Good catch 😄

Reply:
yeah... when checking the test then I realized that I've overlooked on this...